modules/images/image-loading-optimization/class-ilo-grouped-url-metrics.php

```diff
@@ -0,0 +1,295 @@
+<?php
+/**
+ * Image Loading Optimization: ILO_Grouped_URL_Metrics class
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * URL metrics grouped by minimum viewport width for the provided breakpoints.
+ *
+ * @since n.e.x.t
+ * @access private
+ */
+final class ILO_Grouped_URL_Metrics {
```

Comment:
Not a big issue, but I'd prefer to call this `ILO_Grouped_URL_Metrics_Collection` or `ILO_Grouped_URL_Metric_Collection`. Or the other way around, `ILO_URL_Metric_Group_Collection`.

In any case, the "collection" in the name avoids the idea of odd method names like `are_...()` and we could just use `is...()` instead, and it's a common naming pattern for this kind of class.

Reply:
I feel inclined to keep it as `ILO_Grouped_URL_Metrics`. I feel adding `Collection` on the end makes it redundant. I solved the issue with the `are_` method name.

Reply:
I went ahead and renamed it to `ILO_URL_Metrics_Group_Collection` in eb864da9.